main.js

```diff
@@ -148,11 +148,11 @@ function saveOutfit() {
 
 function getOutfitCards(){
   if (window.localStorage.getItem('outfitTitles') === null) {
-    var outfitTitlesParsed = [];
+    outfits = [];
   } else {
-    var outfitTitlesParsed = JSON.parse(window.localStorage.getItem('outfitTitles'));
-    for (var i = 0; i < outfitTitlesParsed.length; i++) {
-      var outfitButton = `<button id="${outfitTitlesParsed[i]}" class="button-style">${outfitTitlesParsed[i]}<img class="close-btn" src="assets/close.svg" alt="Close"></button>`;
+    outfits = JSON.parse(window.localStorage.getItem('outfitTitles'));
+    for (var i = 0; i < outfits.length; i++) {
+      var outfitButton = `<button id="${outfits[i]}" class="button-style">${outfits[i]}<img class="close-btn" src="assets/close.svg" alt="Close"></button>`;
```

Comment:
Just to clarify, what we're doing here is getting rid of the new variable of outfitTitlesParsed and using our outfits array instead, just overwriting it with either a blank array (if nothing in the outfitTitles stringified array in LS) or parsing the outfitTitles array and overwriting outfits with the contents (then looping through that and creating a card for each title in the array)?

Reply:
Yep! I think the outfitTitlesParsed wasn't actually doing anything. It probably was before when we made that . But after the other changes I think it became irrelevant.